galley/pkg/config/analysis/analyzers/testdata/injection-with-mismatched-sidecar.yaml

```diff
@@ -0,0 +1,33 @@
+# Injector is configured to inject 1.3.1, but we have a running pod with 1.3.0
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: sidecarInjectorWebhook
+  name: istio-sidecar-injector-pod
+  namespace: istio-system
+spec:
+  containers:
+  - image: docker.io/istio/sidecar_injector:1.3.1
+    name: sidecar-injector-webhook
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    istio-injection: enabled
+  name: default
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: details
+  name: details-v1-pod
+  namespace: default
+spec:
+  containers:
+  - image: docker.io/istio/examples-bookinfo-details-v1:1.15.0
+    name: details
+  - image: docker.io/istio/proxyv2:1.3.0
```

Comment:
1. Please annotate the YAML with comments pointing out which examples you expect to fail or succeed. 
2. Please add a negative example (e.g. a pod where the version *does* match, and for which we don't expect any error message).

Reply:
Done!